Services/Saml/classes/class.ilSamlAttributeMapping.php

```diff
@@ -0,0 +1,222 @@
+<?php
+// saml-patch: begin
```

Comment:
Please remove all `// saml-patch` markers. They will confuse somewhere in the future...